runner/run_test.go

```diff
@@ -472,11 +490,11 @@ func TestOverrideRun(t *testing.T) {
 	}
 	replaceDestinationInTest(&ftwTest, *fakeDestination)
 
-	t.Run("override and execute all", func(t *testing.T) {
```

Comment:
There were some `t.Run` in tests with only one case which is redundant so removed them

Reply:
Cool!